atk.yml

```diff
@@ -71,6 +71,21 @@ services:
         PIP_REQUIREMENTS: "${DEFAULT_PIP_REQUIREMENTS}"
         ROSDEP_METAPACKAGE: "art_art5_meta"
         ROSDEP_SKIP_KEYS: "python3-torchvision tf_transformations"
+  oak:
+    extends: dev
```

Comment:
Can you create a common `agx` service that isn't meant to be run as a service, but instead has all the shared elements from this and `art5` (and the future other agx services). Does this make sense? Like really the only new stuff across each unique agx service is the `image`, `hostname`, `container_name`, and `ROSDEP_METAPACKAGE`.

Reply:
yessir